### PR TITLE
[yamahareceiver] Avoid multiple async calls when updating the zone information

### PR DESCRIPTION
**Pull request by lolodomo**
Potential fix for #7184

Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

**Comment by lolodomo** [comment]
With this change, all the commands to refresh the zones informations will now be run by the briodge handler in sequence by ther refresh job in one unique thread (every 60 seconds per default).
When the thing handler initializes, this is run in a separate thread.

Previously, the refresh job started multiple threads in parallel, one for each command to run.

**Comment by lolodomo** [comment]
@zarusz : can you check the result of this change ? It has to be tested.

File: bundles/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/handler/YamahaZoneThingHandler.java [hunk 1]
```diff
@@ -236,23 +236,23 @@ boolean isCorrectlyInitialized() {
...
 
         if (inputWithPresetControl != null) {
-            updateAsyncMakeOfflineIfFail(inputWithPresetControl);
+            updateMakeOfflineIfFail(inputWithPresetControl);
         }
 
         if (inputWithDabBandControl != null) {
-            updateAsyncMakeOfflineIfFail(inputWithDabBandControl);
+            updateMakeOfflineIfFail(inputWithDabBandControl);
         }
```
Comment (cpmeister) [inline bundles/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/handler/YamahaZoneThingHandler.java:256]: Perhaps you could allow the updateMakeOfflineIfFail param to be nullable and just exit early if it is null. That should allow you to get rid of a bunch of these null checks in `updateZoneInformation`.

In addition, you would be able to do something like this:
```
Stream.of(zoneAvailableInputs, zoneControl, inputWithPlayControl, inputWithNavigationControl, inputWithPresetControl, inputWithDabBandControl)
.forEach(this::updateAsyncMakeOfflineIfFail);
```

WDYT?

Comment (lolodomo) [reply bundles/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/handler/YamahaZoneThingHandler.java:256]: I think it is a good idea.

**Comment by lolodomo** [comment]
I submitted a new PR to resolve the conflict but I took into accounts all review comments.

@cpmeister : don't merge immediately, I would like to give a chance to the binding maintener to test the change and confirm that nothing is broken.

**Comment by lolodomo** [comment]
I kept unchanged the other calls to updateAsyncMakeOfflineIfFail, they will happen after handling a command that impacts the receiver input. But I don't really understand the need to run these commands asynchronously.
Waiting for @zarusz explanation.

**Comment by lolodomo** [comment]
The feedback is that this change is useless so I close it.

**Pull request closed without merging**